applications/tari_console_wallet/src/config.rs

```diff
@@ -20,12 +20,15 @@
 //  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#![allow(dead_code, unused)]
```

Comment:
I think the problem here is that we have a lib and a bin in the same crate. 
The bin is including the modules directly e.g. `mod automation` and also uses the lib `use tari_console_wallet`. The former module is built into the bin and unused. 

I think we'll need to expose the modules via the `tari_console_wallet` lib and have the main/bin use that.